src/app/containers/RadioPageBlocks/Blocks/LiveRadio/index.jsx

```diff
@@ -11,6 +11,9 @@ import { RequestContext } from '#contexts/RequestContext';
 import { ServiceContext } from '#contexts/ServiceContext';
 import embedUrl from '../../../MediaPlayer/helpers/embedUrl';
 
+const liveRadioPlaceholderImageSrc =
+  'https://news.files.bbci.co.uk/include/articles/public/images/audio-player-placeholder.png';
```

Comment:
Potentially a stupid question but should we be putting this image through iChef to ensure the most optimal size is returned?

Reply:
For it to go through ichef it needs to be published on one of the stacks ichef knows about. Given we want this fixed quickly just using the single static image and allowing the browser to stretch it should be fine. 

We can look to improve this when UX provide us an asset (this image is currently one I've generated temporarily), and then we can evaluate what this placeholder for AMP should look like on mobile widths.